ee/server/service/hosts.go

```diff
@@ -63,7 +70,26 @@ func (svc *Service) RunHostScript(ctx context.Context, request *fleet.HostScript
 		return nil, fleet.NewInvalidArgumentError("script_contents", fmt.Sprintf("script is too long, must be at most %d characters", maxScriptRuneLen))
 	}
 
-	// TODO(mna): any other validation we want to apply to the script? What is the "must be bash/powershell" check?
+	// script must be a "text file", but that's not so simple to validate, so we
```

Comment:
Typically this would be something configured server-wide (you often see this in setups that use a reverse proxy like nginx or apache, or even caddy: https://caddyserver.com/docs/caddyfile/directives/request_body#syntax). I'm guessing something similar must be available in cloud web server frontends.

I don't think we have such a limit configured in our code on the Go http server (we do have a size limit set for headers: https://github.com/fleetdm/fleet/blob/main/server/config/config.go#L99-L112

That being said, it's a potential issue for every endpoint, I think if we want to address that in fleet (vs some web frontend in deployments), that should be done at the server-wide level. Good point that we should probably have at least a fallback, very large but not ridiculously so, limit set that folks can override if needed.